modular_skyrat/code/_HELPERS/roundend.dm

```diff
@@ -7,6 +7,8 @@
 		m.client.verbs += /client/proc/eorg_teleport
 		if (m.client && m.client.prefs && m.client.prefs.eorg_teleport)
 			var/eorg_tele_loc = pick(GLOB.eorg_teleport.loc)
+			if (!isobserver(m))
```

Comment:
You do have to remove the ! now heh, now it _only_ teleports observers.

Also, I once again suggest only taking people from the shuttle transit and centcomm z-level. Zapping out people just RPing and interacting on the station has consistently proven to be a massive fucking pain. "Oh, I forgot I had that setting on ;(" has happened too often.

Reply:
Oh yeah x_x
_I only get like 4 hours of sleep between shifts it's showing_

With the whole dragging people from everywhere, it's more down to everywhere else being an EORG-allowed area. I'll try and come up with a solution for that though.